process/lifecycle.md

```diff
@@ -9,101 +9,68 @@ has_children: true
-* What roles does the project have (e.g. maintainers, committers?) Who are the current core committers of the project, or where can a list of committers be found?
-* What mailing lists are currently used by the project?
-* What tool or platform is leveraged by the project for issue tracking?
-* Does the project have a Core Infrastructure Initiative security best practices badge? Do you foresee any challenges obtaining one? (See: https://bestpractices.coreinfrastructure.org)
-* What is the project's website? Is there a wiki?
-* What social media accounts are used by the project?
-* What is the project’s release methodology and cadence?
-* Are any trademarks, registered or unregistered, leveraged by the project? Have any trademark registrations been filed by the project or any third party anywhere in the world?
-
-## TAC presentation
-
-Projects will present at the next TAC meeting no sooner than two weeks after their submission date.
-
-Proposed projects will receive a 30-45 minute presentation slot at an upcoming TAC meeting as the schedule allows. The project proposal must include a presentation that conforms to the presentation structure below to ensure uniform and complete submissions.
-
-*   Overview of the project and its purpose
-    *  If a project is a new project, it’s often helpful to share the vision for the project, the anticipated structure and benefits.
-    *  If the project is an existing open source project proposed to join ASWF, it is often helpful to schedule a project walkthrough for the community to better understand the project, including the architecture, structure and how to get started using the project. This often helps reviews progress more efficiently.
-*   How does this submission support the ASWF Mission and Vision Statements?
-*   Does the project have any users? 
-    *   How do you plan to attract users if accepted?
-*   How many committers are you planning to have upfront, and from which companies? 
-    *   How do you plan to attract committers and contributors if accepted?
-*   Architecture (functional / non-functional aspects), design, feature overview
-*   Demo and/or walk through of the project
-
-An initial review of a proposal submitted to the TAC should be conducted within two to four weeks following acceptance of a project contribution proposal. 
-
-The TAC may consider the project for approval at the proposed stage during the meeting it is presented at, provided there is a quorum of TAC voting members present. If there is not a quorum of TAC voting members present or if the TAC is not ready to consider the project for approval during the meeting, the TAC may consider discussing and/or voting on the project proposal via email or postpone the vote to a future meeting. An affirmative majority vote of the TAC is required to accept a new TAC Project.
+The Academy Software Foundation may adopt or amend this document by votes of its Technical Advisory Council (“TAC”) and Governing Board.
```

Comment:
Should this be explicit as to voting conditions? Simple majority OK? It's also ambiguous as to whether this requires a votes from TAC, Governing Board, both?